app/controllers/settings_controller.rb

```diff
@@ -4,7 +4,7 @@ class SettingsController < ApplicationController
   def edit_profile
   end
 
-  def update_profile
+  def update_ldap
```

Comment:
Assignment Branch Condition size for update_ldap is too high. [20.2/20]<br>Method has too many lines. [11/10]